Readme.md

```diff
@@ -3,15 +3,30 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T802166)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Index.razor](./CS/DataGridShowDeleteConfirmation/Pages/Index.razor)
-* [_Host.cshtml](./CS/DataGridShowDeleteConfirmation/Pages/_Host.cshtml)
-<!-- default file list end -->
+# Grid for Blazor - How to display a custom confirmation dialog 
 
-### Blazor Data Grid - How to implement the delete confirmation feature
+The example uses the [DxPopup](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxPopup) to create a custom confirmation dialog that asks user to approve record deleting.
 
-The Data Grid was moved to maintenance support mode. No new features/capabilities will be added to this component. We recommend that you [migrate](https://docs.devexpress.com/Blazor/403162/grid/migrate-from-data-grid-to-grid) to the [Grid](https://docs.devexpress.com/Blazor/403143/grid) component. 
+![Display a custom confirmation dialog before deleting a grid record](application-page.png)
 
-This example illustrates how to implement the confirmation dialog before deleting a record.
+This example demonstrates how to do the following:
+
+- Create a custom confirmation dialog with the **Confirm** and **Cancel** [buttons](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxButton).
+- Add a custom **Delete** button to a [DxGrid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) column.
+- Display the confirmation dialog on the **Delete** button [click](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxButton.Click).

```

Comment:
I noticed on line 18 (below) that you specify that a "user"clicks the button. Does a user click the Delete button in this case as well? If so, can we update this line?

Reply:
You are right, we should update this line as follows:
```suggestion
- Display the confirmation dialog when a user [clicks](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxButton.Click) the **Delete** button.

```